packages/lit-html/src/lit-html.ts

```diff
@@ -1373,11 +1375,11 @@ type EventListenerWithOptions = EventListenerOrEventListenerObject &
  */
 export class EventPart extends AttributePart {
   readonly type = EVENT_PART;
-  private _eventContext?: unknown;
+  private _host?: unknown;
```

Comment:
remove this in favor of `this.options.host`

Reply:
Done.